### DISCUSSION TITLE
Welcome to PowerModelsDistribution.jl Discussions!

### DISCUSSION THREAD

**Discussion by pseudocubic** [lanl-ansi/PowerModelsDistribution.jl#326] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanderclaeys** [comment]
Hey all!

My name is Sander, and I am a PhD student at the University of Leuven. I discovered PMD about two years ago. Funnily enough, through an issue I opened, I got back in touch with @frederikgeth, who left Leuven for a position in Australia a while before.  Since then, through this community, I met other researchers such as @pseudocubic and @ccoffrin. From them, I learned a lot about how to develop maintainable software in a collaborative setting. Apparently there is more to it than writing a script and executing it...

Working with all of them on this package has been one of the most joyful experiences of my PhD, so I would strongly encourage you to get involved as well :)

Cheers,
Sander

**Comment by MartaVanin** [comment]
Hello everyone,
I am Marta and I am a PhD student at the KU Leuven. My work is on decision support to distribution system operators and I have been using PowerModelsDistribution for my research since 2018. I was introduced to the InfrastructureModels-related packages by @frederikgeth and @hakanergun, and I think they are a great resource! I try to contribute to PMD a bit, opening issues or with (so far small) commits/PRs, when I have the time. I also benefit of @sanderclaeys's insights on PMD from time to time, since he is a colleague and (often) in the same time zone.

**Comment by hakanergun** [comment]
Hello everyone,

I am Hakan, a research expert at KU Leuven working on optimisation modelling of transmission and distribution networks. We came across the InfrastructureModels ecosystem a few years ago. Starting with a small PR to @ccoffrin for PowerModels.jl, we found the easiness and effectiveness of developing optimisation models in this ecosystem. I had also the pleasure of meeting @pseudocubic and @ccoffrin in person and exchange ideas on the very first steps of PMD. We use the InfrastructureModels ecosystem very actively in our research and teaching, and try to contribute to the ecosystem as much as we can.  

I hope that we can grow this community and build a solid basis for collaborative research in power networks.
